tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -29,459 +29,467 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
+									})
+
+								})
+							})
+						})
+
+						When("a service is created from the output of the dryRun command with a specific name", func() {
+
+							var name string
+							var svcFullName string
+							JustBeforeEach(func() {
+								name = helper.RandString(6)
+								svcFullName = strings.Join([]string{"EtcdCluster", name}, "/")
+								helper.CmdShouldPass("odo", "service", "create", "--from-file", fileName, name, "--project", commonVar.Project)
+							})
+
+							JustAfterEach(func() {
+								helper.CmdShouldPass("odo", "service", "delete", svcFullName, "-f")
+							})
+
+							When("odo push is executed", func() {
+
+								JustBeforeEach(func() {
+									helper.CmdShouldPass("odo", "push")
+								})
+
+								JustAfterEach(func() {
+								})
+
+								It("should fail to create a service again with the same name", func() {
+									stdOut = helper.CmdShouldFail("odo", "service", "create", "--from-file", fileName, name, "--project", commonVar.Project)
+									Expect(stdOut).To(ContainSubstring("please provide a different name or delete the existing service first"))
+								})
+
+								It("should create pods in running state", func() {
+									// now verify if the pods for the operator have started
+									pods := oc.GetAllPodsInNs(commonVar.Project)
+									// Look for pod with example name because that's the name etcd will give to the pods.
+									etcdPod := regexp.MustCompile(name + `-.[a-z0-9]*`).FindString(pods)
+
+									ocArgs := []string{"get", "pods", etcdPod, "-o", "template=\"{{.status.phase}}\"", "-n", commonVar.Project}
+									helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
+										return strings.Contains(output, "Running")
+									})
+
+								})
+							})
+						})
+					})
+				})
+
+				When("an EtcdCluster instance is created with no name", func() {
+					var stdOut string
+					JustBeforeEach(func() {
+						stdOut = helper.CmdShouldPass("odo", "service", "create", fmt.Sprintf("%s/EtcdCluster", etcdOperator), "--project", commonVar.Project)
+					})
+
+					JustAfterEach(func() {
+					})
+
+					It("should create the EtcdCluster instance", func() {
+						Expect(stdOut).To(ContainSubstring("Successfully added service to the configuration"))
+					})
+
+					It("should insert service definition in devfile.yaml", func() {
+						devfilePath := filepath.Join(commonVar.Context, "devfile.yaml")
+						content, err := ioutil.ReadFile(devfilePath)
+						Expect(err).To(BeNil())
+						matchInOutput := []string{"kubernetes", "inlined", "EtcdCluster", "example"}
+						helper.MatchAllInOutput(string(content), matchInOutput)
+					})
+
+					When("odo push is executed", func() {
+
+						var etcdPod string
+
+						JustBeforeEach(func() {
+							helper.CmdShouldPass("odo", "push")
+						})
+
+						JustAfterEach(func() {
+						})
+
+						It("should create a pod in the namespace in Running state", func() {
```

Comment:
What do you think about changing this description to "should create pods in running state" so that it is in sync with other tests?
For example - 
https://github.com/openshift/odo/pull/4770/files#diff-d2a35ca2af5a1450857f9d133a13691f97b1731ac9fc6fa0c27f03f139805f46R194

https://github.com/openshift/odo/pull/4770/files#diff-d2a35ca2af5a1450857f9d133a13691f97b1731ac9fc6fa0c27f03f139805f46R237

Reply:
changed